.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     uses: babylonlabs-io/.github/.github/workflows/reusable_go_lint_test.yml@v0.11.2
     with:
      run-unit-tests: true
-     run-integration-tests: true
+     run-integration-tests: false
      integration-tests-command: |
       make test-e2e
```

Comment:
Hmm I'd say this is still needed as `make test-e2e` uses `ibctesting` for testing everything locally. You could check the Makefile for more detail. Probably e2e is no longer an accurate word for this test 😅

Reply:
These tests are empty AFAIK.